Unable to install the thanos mixin

**What happened**:
When trying to install the mixin using the command 

```bash
jb install github.com/thanos-io/thanos/mixin/thanos@master          
```

It fails with an error

```
vendor/github.com/thanos-io/thanos/mixin/thanos: no such file or directory
```

Details logs with trace can be found below

**What you expected to happen**:

The mixin should be installed successfully.

**How to reproduce it (as minimally and precisely as possible)**:

Simply
```bash
jb install github.com/thanos-io/thanos/mixin/thanos@master          
```

**Full logs to relevant components**:

<details>Logs
<p>

```bash

GET https://github.com/thanos-io/thanos/archive/d33cb7b629fc0eaf7d51cb480e3f40ccad31678a.tar.gz 200
panic: rename /Users/ahmed/kubernetes/mixin/vendor/.tmp/6a0b6008a3db1f7129a622bc3c843b6d368222728/mixin/thanos /Users/ahmed/kubernetes/monitoring/mixin/vendor/github.com/thanos-io/thanos/mixin/thanos: no such file or directory

goroutine 1 [running]:
github.com/jsonnet-bundler/jsonnet-bundler/pkg.(*GitPackage).Install(0xc000010278, 0x149acc0, 0xc00001c1f0, 0xc000020720, 0x28, 0xc000358300, 0x3e, 0xc00001cf18, 0x6, 0x0, ...)
	/Users/ahmed/Workspace/go/src/github.com/jsonnet-bundler/jsonnet-bundler/pkg/git.go:228 +0x1e9d
github.com/jsonnet-bundler/jsonnet-bundler/pkg.download(0xc00007ea50, 0x0, 0xc00001cf18, 0x6, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc000358300, ...)
	/Users/ahmed/Workspace/go/src/github.com/jsonnet-bundler/jsonnet-bundler/pkg/packages.go:290 +0x1b6
github.com/jsonnet-bundler/jsonnet-bundler/pkg.ensure(0xc00017e7e0, 0xc000358300, 0x3e, 0xc00017f680, 0x3e, 0x0, 0x1)
	/Users/ahmed/Workspace/go/src/github.com/jsonnet-bundler/jsonnet-bundler/pkg/packages.go:234 +0x6d7
github.com/jsonnet-bundler/jsonnet-bundler/pkg.Ensure(0xc00017e7e0, 0x1, 0xc000358300, 0x3e, 0xc00017f680, 0x0, 0x0, 0x0)
	/Users/ahmed/Workspace/go/src/github.com/jsonnet-bundler/jsonnet-bundler/pkg/packages.go:55 +0x67
main.installCommand(0xc000016044, 0x37, 0x141e265, 0x6, 0xc000060fd0, 0x1, 0x1, 0x0, 0xc0001310e0)
	/Users/ahmed/Workspace/go/src/github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb/install.go:76 +0x765
main.Main(0xc0000220b8)
	/Users/ahmed/Workspace/go/src/github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb/main.go:86 +0xc69
main.main()
	/Users/ahmed/Workspace/go/src/github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb/main.go:39 +0x25
```

</p>
</details>

**Environment**:
- OS (e.g. from /etc/os-release): MacOS
- Kernel (e.g. `uname -a`): Darwin MacBook-Pro.local 19.5.0 Darwin Kernel Version 19.5.0: Tue May 26 20:41:44 PDT 2020; root:xnu-6153.121.2~2/RELEASE_X86_64 x86_64
- Go Version: go1.15 darwin/amd64

--- a/mixin/README.md
+++ b/mixin/README.md
@@ -35,7 +35,7 @@ To install:
 go get github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb
 ```
 
-> An e.g. of how to install a given version of this library: `jb install github.com/thanos-io/thanos/mixin/thanos@master`.
+> An e.g. of how to install a given version of this library: `jb install github.com/thanos-io/thanos/mixin@master`.
 
 ## Use as a library
 
@@ -44,7 +44,7 @@ To use the `thanos-mixin` as a dependency, simply use the `jsonnet-bundler` inst
 $ mkdir thanos-mixin; cd thanos-mixin
 $ jb init  # Creates the initial/empty `jsonnetfile.json`
 # Install the thanos-mixin dependency
-$ jb install github.com/thanos-io/thanos/mixin/thanos@master # Creates `vendor/` & `jsonnetfile.lock.json`, and fills in `jsonnetfile.json`
+$ jb install github.com/thanos-io/thanos/mixin@master # Creates `vendor/` & `jsonnetfile.lock.json`, and fills in `jsonnetfile.json`
 ```
 
 To update the `thanos-mixin` as a dependency, simply use the `jsonnet-bundler` update functionality:

